schema/release-schema.json

```diff
@@ -1378,7 +1378,7 @@
           "description": "(Deprecated in 1.1. Use transaction.payee instead). The Organization Identifier for the organization which receives the funds in this transaction. Expressed following the Organizational Identifier standard - consult the documentation and the codelist.",
           "$ref": "#/definitions/Identifier",
           "deprecated": {
-            "description": "This field has been replaced by the `transaction.payee` field to resolve ambiguity arising from 'receiver' being interpreted as relating to the goods or services procured rather than the flow of funds between the parties.",
+            "description": "This field has been replaced by the `transaction.payee` field to resolve ambiguity arising from 'receiver' being interpreted as relating to the goods , services and works procured rather than the flow of funds between the parties.",
```

Comment:
Was the conjunction changed from 'or' to 'and' deliberately?

Reply:
yes, as previous comment

Reply:
Per https://github.com/open-contracting/standard/pull/1530#discussion_r908510347, let's keep 'or'